app/src/main/res/layout/fragment_we_are_hiring.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/we_are_hiring" />
+
+    <Button
+        android:id="@+id/apply_now"
+        style="@style/Button"
+        android:layout_width="0dp"
+        android:layout_height="48dp"
```

Comment:
height of the button should be a constant dimension, could you add to dimens.xml something like `default_button_height`?